app/server-utils/stampy.ts

```diff
@@ -256,14 +255,15 @@ const toTag = (r: CodaRow, nameToId: Record<string, string>): Tag => ({
     .filter((q) => q.pageid),
   mainQuestion: extractMainQuestion(r.values['Main question']),
 })
-export const loadAllTags = withCache('allTags', async () => {
```

Comment:
the tags -> questions used to work by first fetching all tags and questions, then making a big, global map for them. This will now fetch each tag separately once it gets clicked on. It's slower (at least the first time a tag is displayed), but is less magical